### ISSUE TITLE
java.lang.Exception: sass4clj.webjars doesn't know how to handle "file" urls

### ISSUE THREAD

**Issue by pandeiro** [Deraen/sass4clj#8]
Using the `sass` task in Boot, I'm getting this exception whenever I make any changes to any `.scss` file and compilation is broken until the pipeline is restarted. Using `deraen/boot-sass "0.1.1"` along with                      `org.webjars.bower/bootstrap-sass "3.3.6"` and `org.webjars.bower/compass-mixins "0.12.7"`.

**Comment by Deraen** [comment]
How about with version `0.2.0`?

**Comment by pandeiro** [comment]
I've tried just now @Deraen, but with `0.2.0` compilation is hanging at "Compiling {sass}... 7 changed files." indefinitely, seemingly breaking the Boot `watch` pipeline entirely (but silently).

I've just run the pipeline with `-vv`, but I don't see anything that seems directly related, just mostly boot-cljs output. Is there anything I should be looking for?

**Comment by Deraen** [comment]
Have you tried without compass-mixins?

**Comment by pandeiro** [comment]
I just tried without the compass-mixins package and observed the same behavior. Is there something within that package that might be problematic though?

**Comment by pandeiro** [comment]
Here's the full stacktrace as shown in Boot with sass4clj 0.1.1, just for reference:

```
Compiling {sass}... 1 changed files.
java.lang.Exception: sass4clj.webjars doesn't know how to handle "file" urls
        sass4clj.webjars/list-assets/fn         webjars.clj:   25
              clojure.core.protocols/fn       protocols.clj:  167
            clojure.core.protocols/fn/G       protocols.clj:   19
      clojure.core.protocols/seq-reduce       protocols.clj:   31
              clojure.core.protocols/fn       protocols.clj:   95
            clojure.core.protocols/fn/G       protocols.clj:   13
                    clojure.core/reduce            core.clj: 6519
           sass4clj.webjars/list-assets         webjars.clj:   14
             sass4clj.webjars/asset-map         webjars.clj:   37
             sass4clj.core/sass-compile            core.clj:  109
     sass4clj.core/sass-compile-to-file            core.clj:  140
                                    ...                          
                     clojure.core/apply            core.clj:  630
                  boot.pod/eval-fn-call             pod.clj:  294
                      boot.pod/call-in*             pod.clj:  315
                                    ...                          
                      boot.pod/call-in*             pod.clj:  318
     deraen.boot-sass/eval6795/fn/fn/fn       boot_sass.clj:   51
     adzerk.boot-cljs/eval2081/fn/fn/fn       boot_cljs.clj:  201
     adzerk.boot-cljs/eval2030/fn/fn/fn       boot_cljs.clj:  135
adzerk.boot-reload/eval2160/fn/fn/fn/fn     boot_reload.clj:  119
   adzerk.boot-reload/eval2160/fn/fn/fn     boot_reload.clj:  118
adzerk.boot-cljs-repl/eval6920/fn/fn/fn  boot_cljs_repl.clj:  197
         boot.task.built-in/fn/fn/fn/fn        built_in.clj:  312
         boot.task.built-in/fn/fn/fn/fn        built_in.clj:  310
   boot.task.built-in/fn/fn/fn/fn/fn/fn        built_in.clj:  264
      boot.task.built-in/fn/fn/fn/fn/fn        built_in.clj:  264
         boot.task.built-in/fn/fn/fn/fn        built_in.clj:  261
                    boot.core/run-tasks            core.clj:  794
                      boot.core/boot/fn            core.clj:  804
    clojure.core/binding-conveyor-fn/fn            core.clj: 1916
```

**Comment by pandeiro** [comment]
Issue in fact _was_ resolved by updating to 0.2.0, once maven dependencies were properly resolved. Closing.
